docker/dockerfiles/Dockerfile.onnx.gpu.dev

```diff
@@ -58,9 +58,6 @@ RUN python3 -m pip install packaging==24.1  && rm -rf ~/.cache/pip
 # Install flash_attn required for Paligemma and Florence2
 RUN python3 -m pip install -r requirements.pali.flash_attn.txt --no-build-isolation && rm -rf ~/.cache/pip
 
-FROM scratch
-COPY --from=base / /
-
```

Comment:
There are other docker files where these were not removed, do we want to have only this single image without COPY all layer?